src/hooks/useHashParams.ts

```diff
@@ -0,0 +1,15 @@
+export default function () {
+  const url = window.location.href
+  const regExStore = /store=(\w+|\w+\d+)/
+  const regExId = /id=(\d+)/
+
+  const matchStore = url.match(regExStore)
+  const matchId = url.match(regExId)
+
+  if (!(matchStore || matchId)) return undefined
+
+  return {
+    hashStore: matchStore && matchStore[1],
+    hashId: matchId && matchId[1],
```

Comment:
You don't need to check for `matchId` here, you did it in your `if` statement, if the `()` doesn't work, simply remove them

Reply:
Changed too